app/lib/utils/translations/translations_transaction.dart

```diff
@@ -0,0 +1,132 @@
+  @override
+  String get unregisterParticipantNotificationBody =>
+      'Your registration for the next cycle has been cancelled. If you change your mind, you can register again.';
+  @override
+  String get unregisterParticipantNotificationTitle => 'Registration Cancelled';
+}
+
+class TranslationsDeTransaction implements TranslationsTransaction {
+  @override
+  String get endorseNewcomerNotificationBody => 'Danke für das Endorsen des Newbies!';
+  @override
+  String get endorseNewcomerNotificationTitle => 'Newbie Endorsed';
+  @override
+  String get registerParticipantNotificationBody => 'Du erhältst eine Erinnerung einen Tag zuvor.';
+  @override
+  String get registerParticipantNotificationTitle => 'Für den nächsten Cycle angemeldet!';
+  @override
+  String get attestNotificationBody =>
+      'Wenn alle Teilnehmer die Attestierungen geschickt haben, kannst du versuchen das Einkommen zu erhalten.';
+  @override
+  String get attestNotificationTitle => 'Teilnehmer attestiert';
+  @override
+  String get claimRewardsNotificationBody => 'Du hast dein Gemeinschaftseinkommen bereits erhalten.';
+  @override
+  String get claimRewardsNotificationTitle => 'Gemeinschaftseinkommen angefordert';
+  @override
+  String get balanceTransferNotificationBody => 'Der Empfänger hat die Tokens erhalten.';
+  @override
+  String get balanceTransferNotificationTitle => 'Transaktion abgeschlossen';
+  @override
+  String get unregisterParticipantNotificationBody =>
+      'Ihre Anmeldung für den nächsten Zyklus wurde storniert. Wenn Sie es sich anders überlegen, können Sie sich erneut anmelden.';
+  @override
+  String get unregisterParticipantNotificationTitle => 'Registrierung storniert';
+}
+
+class TranslationsFrTransaction implements TranslationsTransaction {
+  @override
+  String get endorseNewcomerNotificationBody => "Merci d'avoir endossé le Novice!";
+  @override
+  String get endorseNewcomerNotificationTitle => 'Novice endossé';
+  @override
+  String get registerParticipantNotificationBody => 'Tu recevras un rappel un jour avant.';
+  @override
+  String get registerParticipantNotificationTitle => 'Inscrit pour le prochain cycle!';
+  @override
+  String get attestNotificationBody =>
+      "Une fois que tous les participants ont envoyé les attestations, tu peux essayer d'obtenir le revenu.";
+  @override
+  String get attestNotificationTitle => 'Participants attestés';
+  @override
+  String get claimRewardsNotificationBody => 'Tu as déjà reçu ton revenu communautaire.';
+  @override
+  String get claimRewardsNotificationTitle => 'Revenu communautaire demandé';
+  @override
+  String get balanceTransferNotificationBody => 'Le destinataire a reçu les tokens.';
+  @override
+  String get balanceTransferNotificationTitle => 'Transaction complète';
+  @override
+  String get unregisterParticipantNotificationBody =>
+      "Votre inscription pour le prochain cycle a été annulée. Si vous changez d'avis, vous pouvez vous inscrire à nouveau.";
```

Comment:
Same here, it always used the polite form.

```suggestion
      "Tu t'es désinscrit du prochain cycle. Tu peux te réinscrire si tu changes d'avis.";
```